operator/webhook/Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION ?= 0.4.0
+VERSION := $(shell cat version.txt)
```

Comment:
:question: What's the purpose of moving the version out to a separate file? 

If it's just to make it easier to get into CI, an alternative approach would be to have a `make get-version` command to print that out. This strategy might also work if you'd like a simple way to get the docker-registry and image-name from the Makefile.

Reply:
Yea the idea was to use the same version for CI as the Makefile. Having the version in the Makefile with get-version instead of version.txt seems a little more natural so I'll go with that.